response.cpp

```diff
@@ -1,6 +1,6 @@
-//Based off http://www.boost.org/doc/libs/1_62_0/doc/html/boost_asio/example/cpp11/http/server/reply.cpp
+//Based off http://www.boost.org/doc/libs/1_62_0/doc/html/boost_asio/example/cpp11/http/server/Response.cpp
```

Comment:
but the common api calls it Response and unfortunately we have to adhere to that